### PR TITLE
Fix citation information, contact info and broken links in episode 07

### LINKED ISSUE #58: Move "how to cite" to its own page in the extras directory
"**How to cite**" has appeared in 
the[ readme](https://github.com/ESMValGroup/ESMValTool_Tutorial#citation), 
[Index](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/index.md#how-to-cite-esmvaltool),
[introduction](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/_episodes/01-introduction.md#what-is-esmvaltool), last line in the section what is esmvaltool.
[episode7-conclusion](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/_episodes/07-conclusions.md#how-do-i-cite-esmvaltool) 
and[ about page](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/_extras/about.md#citation). 

It appears too many places. We should just add it in an extras page, and link to it in those other places.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- ~~[ ] All code instructions have been tested.~~

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #58 #68 #79 #85

### CODE REVIEW

**Comment by SarahAlidoost** [comment]
Most of the line length linter errors are on the links that cannot be split into several lines.

File: CITATION [hunk 1]
```diff
@@ -6,10 +6,46 @@ Please cite this tutorial as:
...
+  Andrews, O., Cionni, I., Davin, E. L., Deser, C., Ehbrecht, C., Friedlingstein, P.,
+  Gleckler, P., Gottschaldt, K.-D., Hagemann, S., Juckes, M., Kindermann, S.,
+  Krasting, J., Kunert, D., Levine, R., Loew, A., Mäkelä, J., Martin, G., Mason, E.,
+  Phillips, A. S., Read, S., Rio, C., Roehrig, R., Senftleben, D., Sterl, A.,
+  van Ulft, L. H., Walton, J., Wang, S., and Williams, K. D.:
+  ESMValTool (v1.0) – a community diagnostic and performance metrics tool for
+  routine evaluation of Earth system models in CMIP,
+  Geosci. Model Dev., 9, 1747–1802,
+  [https://doi.org/10.5194/gmd-9-1747-2016](https://doi.org/10.5194/gmd-9-1747-2016),
+  2016.
```
Comment (bouweandela) [inline CITATION:51]: I'm not sure if we should add this one, it is about version 1 of the tool and that was quite different.

File: CITATION [hunk 2]
```diff
@@ -6,10 +6,46 @@ Please cite this tutorial as:
...
+  Hardiman, S., Hassler, B., Hunter, A., Kadow, C., Kindermann, S., Koirala, S.,
+  Koldunov, N. V., Lejeune, Q., Lembo, V., Lovato, T., Lucarini, V., Massonnet, F.,
+  Müller, B., Pandde, A., Pérez-Zanón, N., Phillips, A., Predoi, V., Russell, J., Sellar, A.,
+  Serva, F., Stacke, T., Swaminathan, R., Torralba, V., Vegas-Regidor, J., von Hardenberg, J.,
+  Weigel, K., and Zimmermann, K.:
+  ESMValTool v2.0 – Extended set of large-scale diagnostics for quasi-operational
+  and comprehensive evaluation of Earth system models in CMIP,
+  Geosci. Model Dev. Discuss.,
+  [https://doi.org/10.5194/gmd-2019-291](https://doi.org/10.5194/gmd-2019-291),
+   in review, 2019.
```
Comment (bouweandela) [inline CITATION:39]:
```suggestion
  [ https://doi.org/10.5194/gmd-13-3383-2020]( https://doi.org/10.5194/gmd-13-3383-2020),
   2020.
```
this paper has been published now, maybe move it above the other one?

Comment (bouweandela) [reply CITATION:39]: You can find an up to date list of papers here: https://www.esmvaltool.org/references.html

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -161,10 +161,9 @@ This address can be opened in your browser to preview the tutorial website.
 
 General discussion of the tutorial happens in
 ESMValTool [User Engagement Team][user-engagement].
-You can reach us by email.
-Please see the [contact information][contact-info].
+You can reach us by [email][email].
 
-[email]: mailto:TODO_FIX_ME
+[email]: mailto:esmvaltool@listserv.dfn.de
```
Comment (bouweandela) [inline CONTRIBUTING.md:166]: I think you may have to subscribe to the mailinglist before you're allowed to send it an email, maybe add a link to https://docs.esmvaltool.org/en/latest/community/contact.html#user-mailing-list or explain this?

File: _episodes/07-conclusions.md [hunk 4]
```diff
@@ -38,101 +50,45 @@ From here, there are lots of ways that you can continue to use ESMValTool.
...
->   - What will your final figure show? 
+>   - What will your final figure show?
 {: .challenge}
 
 ### Where can I get more information on ESMValTool?
 
-> ## Additional resources: 
+> ## Additional resources:
+>
 > - [Read the docs page](https://esmvaltool.readthedocs.io/)
```
Comment (bouweandela) [inline _episodes/07-conclusions.md:60]:
```suggestion
> - [Documenation](https://docs.esmvaltool.org)
```

File: _episodes/07-conclusions.md [hunk 5]
```diff
@@ -38,101 +50,45 @@ From here, there are lots of ways that you can continue to use ESMValTool.
...
 > - [ESMValTool home page](https://www.esmvaltool.org/)
-> - [Technical description paper](https://doi.org/10.5194/gmd-13-1179-2020)
+> - [Technical description papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```
Comment (bouweandela) [inline _episodes/07-conclusions.md:62]:
```suggestion
> - [Papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```
there is only 1 technical paper, the others describe the scientific analyses made available through the included recipes

File: _episodes/07-conclusions.md [hunk 6]
```diff
@@ -38,101 +50,45 @@ From here, there are lots of ways that you can continue to use ESMValTool.
...
+[email](mailto:esmvaltool@listserv.dfn.de).
 
-### What if I find a bug? 
+### What if I find a bug?
 
-If you find a bug, please report it back to the ESMValTool team. 
-This will help us fix it so that you can continue working, 
-but also it means that ESMValTool will be more stable for everyone else as well. 
+If you find a bug, please report it back to the ESMValTool team.
+This will help us fix it so that you can continue working,
```
Comment (bouweandela) [inline _episodes/07-conclusions.md:81]:
```suggestion
This will help us fix it, so that you can continue working,
```

File: _extras/about.md [hunk 7]
```diff
@@ -24,8 +24,9 @@ ESMValTool is open source and available on github, and is written
 in PEP8-compliant Python3.
 
 For more details on ESMValTool, please go to:
-- [https://esmvaltool.org](ESMValTool home page)
-- [https://esmvaltool.readthedocs.io/](ESMValTool Read The Docs page)
+
+- [ESMValTool home page](https://esmvaltool.org)
+- [ESMValTool Read The Docs page](https://esmvaltool.readthedocs.io/)
```
Comment (bouweandela) [inline _extras/about.md:29]:
```suggestion
- [ESMValTool Documentation page](https://docs.esmvaltool.org)
```

File: index.md [hunk 8]
```diff
@@ -72,28 +73,15 @@ independently.
 >
 > - [Read the docs page](https://esmvaltool.readthedocs.io/)
```
Comment (bouweandela) [inline index.md:74]:
```suggestion
> - [Documentation](https://docs.esmvaltool.org)
```

File: index.md [hunk 9]
```diff
@@ -72,28 +73,15 @@ independently.
...
 > - [ESMValTool home page](https://www.esmvaltool.org/)
-> - [Technical description paper](https://doi.org/10.5194/gmd-13-1179-2020)
+> - [Technical description papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```
Comment (bouweandela) [inline index.md:76]:
```suggestion
> - [Papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (bouweandela) [reply CONTRIBUTING.md:166]: Also in all the other places where the email address is mentioned.

File: CITATION (hunk 2 above)
Comment (SarahAlidoost) [reply CITATION]: @bouweandela thank you. I replaced the additional resources with the link to up to date list of papers. It avoids duplication.

File: CONTRIBUTING.md (hunk 3 above)
Comment (SarahAlidoost) [reply CONTRIBUTING.md]: I fixed it in all places.

**Comment by SarahAlidoost** [comment]
> Looks like a big improvement already! Would it be possible to rename CITATION to CITATION.md, so it renders nicely on GitHub?

@bouweandela thanks. Once I tried this for both CITATION and LICENSE. it will break the template.

File: CITATION (hunk 1 above)
Comment (SarahAlidoost) [reply CITATION]: I replaced it with a link to the reference list suggested by you.

**Comment by bouweandela** [comment]
#102

**Pull request merged**